module/apmfasthttp/server.go

```diff
@@ -65,16 +65,15 @@ func Wrap(handler fasthttp.RequestHandler, options ...ServerOption) fasthttp.Req
 func (h *apmHandler) handler(ctx *fasthttp.RequestCtx) {
 	if !h.tracer.Recording() || h.requestIgnorer(ctx) {
 		h.requestHandler(ctx)
-
 		return
 	}
 
-	tx, bc, err := StartTransactionWithBody(h.tracer, h.requestName(ctx), ctx)
+	tx, bc, err := StartTransactionWithBody(ctx, h.tracer, h.requestName(ctx))
 	if err != nil {
 		ctx.Error(err.Error(), fasthttp.StatusInternalServerError)
-
 		return
 	}
+	defer tx.End()
```

Comment:
```suggestion
```

closer will be called when the response body is fully written, which I think is right in terms of measuring the request duration